spec/domain/audits/find_content_spec.rb

```diff
@@ -45,6 +46,20 @@ module Audits
       end
     end
 
+    describe '#my_content' do
+      subject(:relation) { described_class.my_content(user.uid) }
+
+      before do
+        Audits::Allocation.create(uid: user.uid, content_id: Content::Item.first.content_id)
+        Audits::Allocation.create(uid: user.uid, content_id: Content::Item.second.content_id)
+        Audits::Allocation.create(uid: user.uid, content_id: Content::Item.third.content_id)
+      end
+
+      it 'returns my content' do
+        expect(relation).to match_array(content_items.sort_by(&:id)[0...3])
```

Comment:
Why is the ordering important here? Either don't assert this, or explain it in the `describe` / `it`.

Reply:
👍